### ISSUE TITLE
White screen after docker guac install

### ISSUE THREAD

**Issue by G-man88** [MysticRyuujin/guac-install#57]
I ran the docker-install.sh script from MysticRyuujin it installed the three docker containers needed to run guacamole in docker, but I'm getting a white screen on the guacamole homepage. I know this is caused by the mysql database not being initialized. My problem is I followed the directions on the guacamole site to create the sql script to initialize it but I don't know what to do now that it's created. Can someone help me with this issue?

Thanks,

**Comment by alesc** [comment]
Hello, same problem here ! 
Guacamole 0.9.14 is not compatible with the latest version of mysql image !
To make it work execute this : 

```
docker exec -it guacamole bash -c "
wget https://dev.mysql.com/get/Downloads/Connector-J/mysql-connector-java-5.1.46.tar.gz
tar xf mysql-connector-java-5.1.46.tar.gz
rm mysql-connector-java-5.1.46.tar.gz
mv mysql-connector-java-5.1.46/mysql-connector-java-5.1.46-bin.jar /opt/guacamole/mysql/
rm /opt/guacamole/mysql/mysql-connector-java-5.1.35-bin.jar
"
docker restart guacamole
```

**Comment by G-man88** [comment]
Unfortunately alesc running those commands didn't resolve my problem.  I was under the impression that the issue was that the database hadn't been initialized in mysql. When you run the docker-install.sh script should it initialize the database?

**Comment by alesc** [comment]
did you kept your log ? Sometime database is not ready yet to accept query, initialization is handled by the script. Delete your dockers images and retry ;) 
Paste your log here if there is no improvement.

**Comment by G-man88** [comment]
Which log are you referring to? I'll grab it and post it here, but I'm not sure which one you want.

**Comment by alesc** [comment]
The standard output when you execute docker-install.sh

**Comment by mattskipton** [comment]
I had this problem and downgraded to mysql5 and things are working

-docker run --restart=always --detach --name=mysql --env="MYSQL_ROOT_PASSWORD=$mysqlrootpassword" --publish 3306:3306 mysql
+docker run --restart=always --detach --name=mysql --env="MYSQL_ROOT_PASSWORD=$mysqlrootpassword" --publish 3306:3306 mysql:5

**Comment by G-man88** [comment]
Here is the output of the script after removing all containers and images

Enter a MySQL ROOT Password:
Confirm MySQL ROOT Password:
Passwords don't match. Please try again.

Enter a MySQL ROOT Password:
Confirm MySQL ROOT Password:

Enter a Guacamole User Database Password:
Confirm Guacamole User Database Password:

Hit:1 http://us.archive.ubuntu.com/ubuntu bionic InRelease
Hit:2 http://us.archive.ubuntu.com/ubuntu bionic-updates InRelease
Hit:3 http://security.ubuntu.com/ubuntu bionic-security InRelease
Hit:4 http://ppa.launchpad.net/qbittorrent-team/qbittorrent-stable/ubuntu artful InRelease
Hit:5 http://apt.sonarr.tv master InRelease
Hit:6 http://us.archive.ubuntu.com/ubuntu bionic-backports InRelease
Hit:7 http://download.mono-project.com/repo/ubuntu xenial InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree
Reading state information... Done
Package docker-ce is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'docker-ce' has no installation candidate
--2018-11-29 00:42:51--  http://apache.org/dyn/closer.cgi?action=download&filename=guacamole/0.9.14/binary/guacamole-auth-jdbc-0.9.14.tar.gz
Resolving apache.org (apache.org)... 95.216.24.32, 40.79.78.1
Connecting to apache.org (apache.org)|95.216.24.32|:80... connected.
HTTP request sent, awaiting response... 302 Found
Location: http://apache.cs.utah.edu/guacamole/0.9.14/binary/guacamole-auth-jdbc-0.9.14.tar.gz [following]
--2018-11-29 00:42:52--  http://apache.cs.utah.edu/guacamole/0.9.14/binary/guacamole-auth-jdbc-0.9.14.tar.gz
Resolving apache.cs.utah.edu (apache.cs.utah.edu)... 155.98.64.87
Connecting to apache.cs.utah.edu (apache.cs.utah.edu)|155.98.64.87|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 10686897 (10M) [application/x-gzip]
Saving to: ‘guacamole-auth-jdbc-0.9.14.tar.gz’

guacamole-auth-jdbc-0 100%[=========================>]  10.19M  1.47MB/s    in 10s

2018-11-29 00:43:03 (1018 KB/s) - ‘guacamole-auth-jdbc-0.9.14.tar.gz’ saved [10686897/10686897]

Unable to find image 'mysql:latest' locally
latest: Pulling from library/mysql
a5a6f2f73cd8: Pull complete
936836019e67: Pull complete
283fa4c95fb4: Pull complete
1f212fb371f9: Pull complete
e2ae0d063e89: Pull complete
5ed0ae805b65: Pull complete
0283dc49ef4e: Pull complete
a7e1170b4fdb: Pull complete
88918a9e4742: Pull complete
241282fa67c2: Pull complete
b0fecf619210: Pull complete
bebf9f901dcc: Pull complete
Digest: sha256:b7f7479f0a2e7a3f4ce008329572f3497075dc000d8b89bac3134b0fb0288de8
Status: Downloaded newer image for mysql:latest
5263cdc7661b00f0f58224671c3a59cf3228c2ecf02d397baf256ce81e215a82
Waiting 60 seconds for MySQL to load
./docker-install.sh: line 84: mysql: command not found
./docker-install.sh: line 86: mysql: command not found
Unable to find image 'guacamole/guacd:latest' locally
latest: Pulling from guacamole/guacd
af4b0a2388c6: Pull complete
dcdd9c2ece80: Pull complete
674a18e16f54: Pull complete
0e6dd708fc82: Pull complete
300b33db8291: Pull complete
Digest: sha256:3cb2ad5cfecda2bc3b5383a272d22c106b7e860cebe0600345c57f4b61e4484d
Status: Downloaded newer image for guacamole/guacd:latest
5b0113489e5e07d169057696802e5549a75c2dacdc26a5dda94c5befbc366553
Unable to find image 'guacamole/guacamole:latest' locally
latest: Pulling from guacamole/guacamole
a3ed95caeb02: Pull complete
c06482b548c8: Pull complete
954f07d4dc14: Pull complete
0bd71cf3b59f: Pull complete
12d3f8630eca: Pull complete
d6e16c3ae29a: Pull complete
ea9df2460d2a: Pull complete
66a4ea6d102d: Pull complete
2b6aeca0d35e: Pull complete
3ac1388b912d: Pull complete
67accb67652a: Pull complete
Digest: sha256:fa1cc98182c79be24a1a2f82b97fa65a76ba4bd1ad6a9a3cff7b768700f324ec
Status: Downloaded newer image for guacamole/guacamole:latest
6f1f6981604b366387e6d191c3d91b1bc260861671cece45657e79cacd9844b6

**Comment by G-man88** [comment]
@alesc Thank you so much! I didn't think about the mysql client on the local machine, I thought it was all built into the containers. After I installed the mysql client and ran your original fix for the 0.9.14 latest mysql compatibility fix all is well and fixed. You are awesome, thank you again.
